### ISSUE TITLE
Action fails immediately with "invalid value workflow reference" when I try to use "action-automatic-releases"

### ISSUE THREAD

**Issue by jacklund** [marvinpinto/actions#718]
I'm trying to use your action, but for some reason, Github thinks that it's not a reference to an action in the Marketplace, but a local action. Here's my [rust.yml](https://github.com/jacklund/voynich/blob/main/.github/workflows/rust.yml). It seems to bork on line 42.

I've tried enclosing the name in quotes and without quotes, same error:

![image](https://github.com/marvinpinto/actions/assets/209634/f8f71344-845b-47e1-8647-eac3936743cd)

Has anyone else had this issue? Is it (quite possibly) something dopey I'm doing?

Thanks,

-Jack

**Comment by jacklund** [comment]
I should mention that I've also filed a [support ticket](https://support.github.com/ticket/personal/0/2643308) with Github, so if y'all think this is purely a GitHub issue, feel free to ignore this. :smiley_cat:

**Comment by aryairani** [comment]
@jacklund Your `rust.yml` link is broken so I'm not sure what's in it, but if I squint at the screenshot, it looks like you're using the action as a `job:` when it should be used as a `step:`.

It looks like you've fixed it already though.

**Comment by jacklund** [comment]
Yeah, sorry, I figured it out (sort of, the syntax really doesn't make a 
lot of sense to me, but ¯\_(ツ)_/¯).

Thanks, though!

-Jack
